get_who_theme.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+import pandas as pd
+import requests
+
+
+def main():
+    # Run like `python get_who_theme.py "Overweight and Obesity"`
+    parser = argparse.ArgumentParser()
+    parser.add_argument("theme_name")
+    args = parser.parse_args()
+
+    requested_indicators = pd.read_csv("requestedIndicators.csv")
+
+    theme = get_theme(args.theme_name, requested_indicators)
+    transform_theme(theme)
+    write_excel(theme)
+    write_csv(theme)
+
+
+def get_theme(theme_name: str, requested_indicators: pd.DataFrame) -> pd.DataFrame:
+    theme = pd.DataFrame()
+
+    theme_indicators = requested_indicators[
+        (requested_indicators["Theme"] == theme_name)
+        & (requested_indicators["HostingReference"] == "WHO")
+    ]
+    theme_indicators = theme_indicators.drop_duplicates("IndicatorCodeDS")
+
+    for _, indicator in theme_indicators.iterrows():
+        indicator_data = get_indicator_data(indicator["IndicatorCodeDS"])
+        indicator_values = get_indicator_values(indicator_data, indicator)
+        theme = theme.append(indicator_values)
+
+    return theme
+
+
+def get_indicator_data(indicator_code: str) -> list:
+    api_url = f"https://ghoapi.azureedge.net/api/{indicator_code}"
+    response = requests.get(api_url)
+    response.raise_for_status()
+    return response.json()["value"]
```

Comment:
Because we're only interested in the list of dictionaries, (i.e., the contents of `"value"`), we can just return that directly, instead of the whole response dictionary.